canvas_mgr.py

```diff
@@ -58,7 +58,7 @@ def dump_out(self):
             json.dump(obj, f, ensure_ascii=False, indent=4)
         return self.g_out[:-1]
 
-    def print_own(self, mystr):
+    def print_own(self, mystr): #change the value of self.g_out
```

Comment:
`# Change ...`

Reply:
?

Reply:
Actually I don't understand what does the comment mean.

Reply:
Your comment format is not consistent with my code

Reply:
Ok,I'll fix it

Reply:
Currently I'm thinking about adding more comments to the code and I found a cool [plugin](https://github.com/folke/todo-comments.nvim). Is this kind of comment format acceptable?

Reply:
I mean the first letter should be capitalized and there should be a space

Reply:
The function specification should be like this:
```
def ddd():
    '''
    Specification
    '''

```